### PR TITLE
change twitter handles into links

### PR DESCRIPTION
**Pull request by achasveachas**
I changed all the Twitter handles into links to the Twitter profiles, I feel like that would be helpful to people like me who want to follow other career changers on Twitter :)

I also changed the wording in the instructions for people who might prefer another social medium other than Twitter or who might not be comfortable sharing their contact info in public.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,55 +7,56 @@ These are guest names I gathered in the fall that I plan to reach out to. If you
 Copy the snippet below, add your details and submit a PR!
 
 ```
-| Full Name | @twitterHandle | Previous Career | ❌ | ❌ |
+| Full Name | [social-media-handle](url-to-social-media-profile)* | Previous Career | ❌ | ❌ |
```
Comment (kyleshevlin) [inline README.md]: Please remove the asterisk here. I want this to remain as copy/paste-able as possible and that's just a small bit of unnecessary markup that I think people will forget to delete.

File: README.md [hunk 2]
```diff
@@ -7,55 +7,56 @@ These are guest names I gathered in the fall that I plan to reach out to. If you
...
 ```
+*if you prefer not to share your contact info publicly you can email me directly at [secondcareerdevs@gmail.com](mailto:secondcareerdevs@gmail.com)
```
Comment (kyleshevlin) [inline README.md]: Let's remove the asterisk here as well. Capitalize "if", slap a period on the end and let's just make this a sentence.

**Review by kyleshevlin — CHANGES_REQUESTED** [review]
Please make changes requested in comments. Thanks!

File: README.md (hunk 1 above)
Comment (achasveachas) [reply README.md]: You are right, I didn't think of that.

File: README.md [hunk 3]
```diff
@@ -7,55 +7,56 @@ These are guest names I gathered in the fall that I plan to reach out to. If you
 Copy the snippet below, add your details and submit a PR!
 
 ```
-| Full Name | @twitterHandle | Previous Career | ❌ | ❌ |
+| Full Name | [social-media-handle](url-to-social-media-profile) | Previous Career | ❌ | ❌ |
```
Comment (kyleshevlin) [inline README.md:10]: Awesome! Thanks for the change. 👍

**Comment by achasveachas** [comment]
Changes made.

**Review by kyleshevlin — APPROVED** [review]
I really appreciate the changes.

**Comment by achasveachas** [comment]
Yay! My first OS contribution! 😄

**Comment by kyleshevlin** [comment]
Well done!

**Pull request merged**